reference/src/layout/unions.md

```diff
@@ -38,3 +28,44 @@ contents are.
 Even if the offsets happen to be all 0, there might still be differences in the
 function call ABI.  If you need to pass unions by-value across an FFI boundary,
 you have to use `#[repr(C)]`.
+
+### Layout of `repr(C)` unions
```

Comment:
Should be "repr C" for consistency.

Or change the other section titles here and in the other documents as well.